src/index.ts

```diff
@@ -279,3 +283,13 @@ export default async function app(
 
   return app;
 }
+
+run()
```

Comment:
Temporary, just to run it locally with local dev temporal server. Should be part of our Tilt ecosystem.